src/main/kotlin/com/jetbrains/snakecharm/inspections/quickfix/AddIgnoredElementQuickFix.kt

```diff
@@ -0,0 +1,52 @@
+package com.jetbrains.snakecharm.inspections.quickfix
+
+import com.intellij.codeInsight.template.TemplateBuilderImpl
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.codeInspection.ex.InspectionProfileModifiableModel
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.profile.codeInspection.InspectionProfileManager
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.jetbrains.snakecharm.SnakemakeBundle
+import com.jetbrains.snakecharm.inspections.SmkUnrecognizedSectionInspection
+
+class AddIgnoredElementQuickFix(element: PsiElement) : LocalQuickFixAndIntentionActionOnPsiElement(element) {
+    private val elementName = element.text
```

Comment:
Do we need here `LocalQuickFixAndIntentionActionOnPsiElement(element)` ? Could we use just ` implements LocalQuickFix, LowPriorityAction` ?

Reply:
Is `elementName` is section name? If yes, please use user-friendly variable name naming